go.mod

```
@@ -107,3 +113,5 @@ require (
 	modernc.org/memory v1.8.2 // indirect
 	modernc.org/sqlite v1.34.4 // indirect
 )
+
+replace github.com/ductone/c1-lambda => ../../conductorone/c1-lambda
```

golangci-lint (1.62.2)

local replacement are not allowed: github.com/ductone/c1-lambda

(gomoddirectives)